get-latest-pulsecore.sh

```diff
@@ -21,28 +21,35 @@ if [ ! -d "$BUILDERDIR" ]; then
     exit 1
 fi
 
-if [ -z "$LATEST_FEDORA_RELEASE" ]; then
-    LATEST_FEDORA_RELEASE="$(git ls-remote --heads https://src.fedoraproject.org/rpms/fedora-release | grep -Po "refs/heads/f[0-9][1-9]*" | sed 's#refs/heads/f##g' | sort -g | tail -1)"
-fi
-LATEST_FEDORA_VERREL="$(dnf -q repoquery pulseaudio --disablerepo=* --enablerepo=fedora --enablerepo=updates --releasever="$LATEST_FEDORA_RELEASE" | grep -Po "[0-9][1-9]*\.*[0-9]*\.*[0-9]*-[0-9]*" | sort -V | tail -1)"
+REPO_URL=git://anongit.freedesktop.org/pulseaudio/pulseaudio
 
-LATEST_FEDORA_VERSION="$(echo "$LATEST_FEDORA_VERREL" | cut -d'-' -f1)"
+LATEST_REPO_VERSION="$(git ls-remote --exit-code --refs --tags --sort="v:refname" "$REPO_URL" '*.*' | tail -n1 | cut -d/ -f3 | sed 's/^v//')"
 LATEST_QUBES_VERSION="$(find "$LOCALDIR/pulse" -type d -name "pulsecore-*" | sed "s|$LOCALDIR/pulse/pulsecore-||" | sort -g | tail -1)"
 
-SRC_RPM="pulseaudio-${LATEST_FEDORA_VERREL}.fc${LATEST_FEDORA_RELEASE}.src.rpm"
-SRC_FILE="pulseaudio-${LATEST_FEDORA_VERSION}.tar.xz"
```

Comment:
`exit_updater()` uses this variable, so it needs to be updated too.

Reply:
Looks like this variable is redundant now, removed.